ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -652,6 +643,17 @@ export const RuntimePanel = fp.flow(
   const {hasMicroarrayData} = fp.find({cdrVersionId}, cdrVersionListResponse.items) || {hasMicroarrayData: false};
   const [{currentRuntime, pendingRuntime}, setRequestedRuntime] = useCustomRuntime(namespace);
 
+  // if runtime configuration type is a default, override its config with preset values
+  if (!!currentRuntime && currentRuntime.status === RuntimeStatus.Deleted) {
```

Comment:
nit: don't need `!!` in a predicate context (where it is already going to be coerced to a boolean)